.github/workflows/ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
          echo "postgres:5432:*:postgres:postgres" > ~/.pgpass
          chmod 600 ~/.pgpass
 
-         gem install bundler --version=1.17.3
+         gem install bundler --version=2.2.4
```

Comment:
We've had tons of trouble with bundler 2.x, has that been fixed?

Reply:
I did not have any to troubles other than twiddling the timing of "bundler" and the ENV to ensure it was used in all Docker images